doc/source/ray-overview/installation.rst

```diff
@@ -338,8 +338,6 @@ Images are `tagged` with the format ``{Ray version}[-{Python version}][-{Platfor
      - A specific Ray release, e.g. 1.12.1
    * - nightly
      - The most recent Ray development build (a recent commit from Github ``master``)
-   * - 6 character Git SHA prefix
```

Comment:
since we are here, could you also update the `1.12.1` (2 lines above) into maybe `2.9.3` ?

Reply:
done! also updated to `2.9.3` for other readme files